lib/github_service/commands/add_label.rb

```diff
@@ -1,6 +1,10 @@
 module GithubService
   module Commands
     class AddLabel < Base
+      UNASSIGNABLE = {
+        "jansa/yes" => "jansa/yes?"
+      }
```

Comment:
Note:  For now this is hard coded, but I think moving forward, we should maintain a list elsewhere for this.

That said, I didn't want to blanket block `*/yes` labels since this is intended for `jansa` and up only.